source/administration/devops-command-center.rst

```diff
@@ -28,7 +28,7 @@ Installing Mattermost Incident Management
 
 *For self-managed deployments*
```

Comment:
@lieut-data, I know IM is pre-packaged so just wanted to confirm this is still accurate for self-managed deployments.